pkg/forwarder/domain_forwarder.go

```diff
@@ -102,21 +105,21 @@ func (f *domainForwarder) retryTransactions(retryBefore time.Time) {
 		if !f.blockedList.isBlock(t.GetTarget()) {
 			select {
 			case f.lowPrio <- t:
-				transactionsRetried.Add(1)
-				tlmTxRetried.Inc(f.domain)
+				transactionsRetried.Add(getTransactionEndpointName(t), 1)
```

Comment:
What do you think about introducing a local variable for `getTransactionEndpointName(t)`. It makes more explicit that the same value is used everywhere.

Reply:
It's not the same value. It depends on the `Transaction` `t`.

Reply:
I am speaking about introducing a local variable before line 83.

Reply:
Ah fair enough. Makes sense to me.